xclim/sdba/base.py

```diff
@@ -282,20 +281,20 @@ def apply(
         out.attrs["group_compute_dims"] = dims
         out.attrs["group_window"] = self.window
 
+        # On non reducing ops, drop the constructed window
+        if self.window > 1 and "window" in out.dims:
+            out = out.isel(window=self.window // 2, drop=True)
```

Comment:
Note that drop does not remove the dimension from the array shape. If this is what you want to do, you need to use `squeeze`.

Reply:
Hum... are you sure? I just tested and it doesn't sem to leave it there? I added a test for this.

Reply:
On my end it drops the coordinates, but not the dimension.